tools/install.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+TENSORFLOW_DIRECTORY='../../tensorflow'
+TENSORFLOW_BIN_DIRECTORY="$TENSORFLOW_DIRECTORY/bazel-bin/tensorflow"
+USR_DIRECTORY='../usr'
+
+# set -x
+
+function copy_file() {
+  if [[ -L "$1/$2" ]]; then
+    local target=`readlink "$1/$2"`
+    copy_file $1 $target $3
+    (cd $3; ln -s $target -f -r $2)
+  else
+    cp "$1/$2" $3
+  fi
+}
+
+function fix_tf_header() {
+  cp "$1" "$2"
+  sed -i -e 's#include "'"$3"'tensorflow/c/c_api.h"#include "c_api.h"#g' "$2"
+  sed -i -e 's#include "'"$3"'tensorflow/c/tf_attrtype.h"#include "tf_attrtype.h"#g' "$2"
+  sed -i -e 's#include "'"$3"'tensorflow/c/tf_status.h"#include "tf_status.h"#g' "$2"
+  sed -i -e 's#include "'"$3"'tensorflow/c/c_api_experimental.h"#include "c_api_experimental.h"#g' "$2"
+  sed -i -e 's#include "'"$3"'tensorflow/c/eager/c_api.h"#include "c_api_eager.h"#g' "$2"
+}
+
+function install_header() {
+  echo "Install header: " $1 $2
+  fix_tf_header $1 "$USR_DIRECTORY/lib/swift/linux/x86_64/modulemaps/CTensorFlow/$2" ""
+}
+
+mkdir -p $USR_DIRECTORY/lib/swift/linux
+copy_file $TENSORFLOW_BIN_DIRECTORY libtensorflow.so $USR_DIRECTORY/lib/swift/linux
+copy_file $TENSORFLOW_BIN_DIRECTORY libtensorflow_framework.so $USR_DIRECTORY/lib/swift/linux
+
+mkdir -p $USR_DIRECTORY/lib/swift/linux/x86_64/modulemaps/CTensorFlow
+install_header "$TENSORFLOW_DIRECTORY/tensorflow/c/c_api.h" c_api.h
```

Comment:
Is the plan to eventually have these headers as part of the tensorflow archive?

Reply:
One plan to investigate is to use the headers directly from the tensorflow installation. That would require using them as implementationOnly from inside swift-apis so that they do not need to be exported alongside swfit-apis.